src/Windows/R/Editor/Impl/Commands/RMouseProcessor.cs

```diff
@@ -30,13 +31,20 @@ public sealed class RMouseProcessor : MouseProcessorBase {
                     _shell.Services.GetService<IProcessServices>().Start(url);
                     return;
                 }
-                // If this is a Ctrl+Click or double-click then post the select word command.
-                var command = new SelectWordCommand(_wpfTextView, _wpfTextView.TextBuffer);
-                var o = new object();
-                var result = command.Invoke(typeof(VSConstants.VSStd2KCmdID).GUID, (int)VSConstants.VSStd2KCmdID.SELECTCURRENTWORD, null, ref o);
-                if (result.Result == CommandResult.Executed.Result) {
-                    e.Handled = true;
-                    return;
+                if (!_wpfTextView.Caret.InVirtualSpace) {
+
+                    // If this is a Ctrl+Click or double-click then post the select word command.
+                    var point = _wpfTextView.MapDownToR(_wpfTextView.Caret.Position.BufferPosition);
+                    if (point.HasValue) {
+                        var command = new SelectWordCommand(_wpfTextView, point.Value.Snapshot.TextBuffer);
+                        var o = new object();
+                        var result = command.Invoke(typeof(VSConstants.VSStd2KCmdID).GUID,
```

Comment:
Why do you need to specify `VSConstants.VSStd2KCmdID.SELECTCURRENTWORD` if `SelectWordCommand` already defines it?